intelmq/lib/test.py

```diff
@@ -188,7 +188,7 @@ def log(name, *args, **kwargs):
             return logger
         return log
 
-    def prepare_bot(self, parameters={}, destination_queues=None):
+    def prepare_bot(self, parameters={}, destination_queues=None, prepare_source_queue: bool = True):
```

Comment:
There are also still changes in unrelated parts of the code

Reply:
I don't see any changes compared to the develop branch:

```
$ git status
On branch shadowserver-202207
Your branch is up to date with 'origin/shadowserver-202207'.

nothing to commit, working tree clean
```

```
$ curl -s 'https://raw.githubusercontent.com/certtools/intelmq/develop/intelmq/lib/test.py' > /tmp/test.py
$ diff -u intelmq/lib/test.py /tmp/test.py
$
```